### PR TITLE
fixed k3d cluster status check, prevent proceeding when stage fails, …

### PR DESCRIPTION
**Pull request by Reagan**
This PR fixes k3d cluster status check, prevent proceeding when stage fails, adds documentation so that the deployment steps are displayed as below

````
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⡿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣴⣿⣿⣿⡟⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⣨⠟⠢⣸⣿⣿⣿⣿⠃⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢤⡀⠀⢠⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠘⡅⠀⠀⢙⣿⣿⣿⠟⠁⠀⢤⡠⠤⣄⠤⢤⡀⢀⠤⠠⡄⠀⡤⠤⢼⡇⠠⢤⠀⢀⡤⠤⡄⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠈⠢⣴⣿⣿⣿⡀⠀⠀⠀⢸⠀⠀⡏⠀⢸⡇⢸⠒⠒⠛⠸⡇⠀⢸⡇⠀⢸⠀⢾⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠙⢿⣿⣿⣿⠆⠀⠀⠘⠀⠀⠃⠀⠘⠃⠈⠑⠒⠃⠀⠙⠒⠙⠓⠐⠚⠂⠈⠑⠒⠂⠀⠀⠀⠀⠀⠀⠀⠀
⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠹⠏⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀
                                                                  Installer version cht-k8s-configuration-changes                                                                 ↳ Checking Python3
↳ Checking pip
↳ Installing invoke
↳ Installing PyYAML
↳ Installing requests
↳ Checking values file
↳ Check if helm is installed
↳ Check if kubectl is installed
↳ Check if k3d is installed when environment is set to 'local'
↳ Check if k3d cluster already exists
K3d cluster 'k3s-default' already exists. Skipping creation.
↳ Check if nginx ingress controller is installed
nginx ingress controller already exists. Skipping installation.
```

### CODE REVIEW

**Comment by mrjones-plip** [comment]
Please resubmit changes against master after this PR is merged:
* https://github.com/medic/cht-core/pull/8693

**Comment by henokgetachew** [comment]
Hey @Reagan can you update the ASCI logo to use the CHT logo instead? I'm working on merging your changes.

**Comment by mrjones-plip** [comment]
I updated the logo and simplified the centered text (I kept on thinking my terminal was broken, noticed Henock had it cropped in his screenshot as well).

Now looks like this:

image

**Comment by mrjones-plip** [comment]
I think this needs a rebase/master merged into as it's not passing CI

File: deploy/kubernetes/secrets/cht-couchdb-credentials.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: Secret
+metadata:
+  name: cht-couchdb-credentials
+type: Opaque
+stringData:
+  COUCHDB_PASSWORD: <password-value>
+  COUCHDB_SECRET: <secret-value>
+  COUCHDB_USER: <user-name>
+  COUCHDB_UUID: <uuid-value>
+  COUCH_URL: http://<user-name>:<password-value>@haproxy.cht-dev-namespace.svc.cluster.local:5984/medic
```
[the remark this answers is not in the document]
Comment (henokgetachew) [reply deploy/kubernetes/secrets/cht-couchdb-credentials.yaml:11]: That file is actually not going to be part of this PR when rebase correctly completed. I however agree with the sentiment.

**Comment by henokgetachew** [comment]
@Reagan I have rebased this. There are however a couple things that still need to be done:
1. Changes were made to the file named `deploy-cht` before it was renamed to `cht-deploy`
2. The changes were made to the `deploy` dir before it was moved to be under `scripts` directory.

When both changes are done, this PR comes back to its original size of your original commits. Basically, manually moving the directory and copying the changes to the `cht-deploy` should suffice to get this PR ready for approval. The simpler move would be to just simply delete the deploy dir from the top level and copy your changes to the deploy dir under scripts that's now in here because of the rebase.

**Comment by garethbowen** [comment]
@Reagan Can you have another look at this?

**Comment by garethbowen** [comment]
This is stalled with no response, conflicting with recent deploy work, and it's in a bad state anyway with what looks like a bad merge or bad ignores (1600 files changed). I'm going to close but leave the branch around in case there's anything we can save from here.

**Pull request closed without merging**